lib/jsdom/living/node.js

```diff
@@ -55,6 +55,11 @@ module.exports.clone = function (node, document, cloneChildren) {
       break;
   }
 
+  // https://html.spec.whatwg.org/multipage/scripting.html#script-processing-model
+  if (node._localName === "script") {
```

Comment:
Can you add a [cloningSteps] to HTMLScriptElement-impl instead?

Reply:
ok!